mithril.d.ts

```diff
@@ -918,8 +918,8 @@ declare module _mithril {
 	}
 }
 
-declare var Mithril: _mithril.MithrilStatic;
-declare var m: _mithril.MithrilStatic;
+declare var _mithril: Mithril.Static;
```

Comment:
Would you mind removing this line entirely? Any code that uses this alias could literally fix their usage by running `sed 's/_mithril/Mithril/g'` across their project.

Also, this variable is never exported. I feel it's incorrect to include, since it doesn't reflect reality.